USERS.md

```diff
@@ -665,6 +665,12 @@ Users (Alphabetically)
       L: https://schubergphilis.com/en
       Q: @stimmerman @shoekstra @mbaumann
 
+    * N: SDV Services
+      D: SDV Services uses Cilium to host Wordpress multi-tenant the cloud-native way and also as the CNI for customer Kubernetes clusters.
+      U: CNI, Networking, NetworkPolicy, Hubble, IPAM, Kubernetes services
+      L: https://sdvservices.nl
+      Q: @Sjouke de Vries
+
```

Comment:
Thanks for adding this, can you split this change into another PR for faster review and merge?

Reply:
Done, see https://github.com/cilium/cilium/pull/34746